kernel/kernel.c

```diff
@@ -28,9 +30,15 @@ static void init_descriptor_tables() {
 	idt_init();
 }
 
+static void kernel_init() {
+	pit_init();
+	// TODO: Put all init function here. With pit_init first to get track of time as soon as possible.
+}
+
 void kernel_main(unsigned long multiboot_info_addr) {
 	multiboot_info_t *mbi = (multiboot_info_t *)multiboot_info_addr;
 
+	kernel_init();
 	init_descriptor_tables();
 	VGA_initialize();
 	KBD_initialize();
```

Comment:
Either move all the init function inside the new `kernel_init` function or move out the `pit_init` in the `kernel_main` and delete the `kernel_init` function.

Reply:
done